CRM/Core/I18n.php

```diff
@@ -801,7 +801,7 @@ function ts($text, $params = []) {
     if ($bootstrapReady) {
       // just got ready: determine whether there is a working custom translation function
       $config = CRM_Core_Config::singleton();
-      if (isset($config->customTranslateFunction) and function_exists($config->customTranslateFunction)) {
+      if (isset($config->customTranslateFunction) && !empty($config->customTranslateFunction) && function_exists($config->customTranslateFunction)) {
```

Comment:
Oh I meant the whole thing would just be `if (!empty($config->customTranslateFunction) && function_exists($config->customTranslateFunction)) {` since there's no need to check isset if checking empty, but ok.

Reply:
ok updated to that now @demeritcowboy